src/main/java/io/github/nucleuspowered/nucleus/configurate/typeserialisers/DelayedTimeValueSerialiser.java

```diff
@@ -0,0 +1,129 @@
+    private CronParser cronjCronParser;
+
+    /**
+     * Constructs a DelayedTimeValueSerialiser, initializes the cron parsers
+     */
+    public DelayedTimeValueSerialiser() {
+        this.unixCronParser = new CronParser(CronDefinitionBuilder.instanceDefinitionFor(CronType.UNIX));
+        this.quartzCronParser = new CronParser(CronDefinitionBuilder.instanceDefinitionFor(CronType.QUARTZ));
+        this.cronjCronParser = new CronParser(CronDefinitionBuilder.instanceDefinitionFor(CronType.CRON4J));
+    }
+
+    @Override
+    public DelayedTimeValue deserialize(TypeToken<?> type, ConfigurationNode value) throws ObjectMappingException {
+        if (value.getValue() instanceof String) {
+            String node = value.getString();
+            try {
+                Cron asUnixCron = this.unixCronParser.parse(node);
+                return new DelayedTimeValue(DelayedTimeType.UNIX_CRON, asUnixCron);
+            } catch (Exception e1) {
+            }
+            try {
+                Cron asQuartzCron = this.quartzCronParser.parse(node);
+                return new DelayedTimeValue(DelayedTimeType.QUARTZ_CRON, asQuartzCron);
+            } catch (Exception e1) {
+            }
+            try {
+                Cron asJavaCron = this.cronjCronParser.parse(node);
+                return new DelayedTimeValue(DelayedTimeType.CRON4J_CRON, asJavaCron);
+            } catch (Exception e1) {
+            }
+            try {
+                WhenTime asWhenTime = WhenTime.fromString(node);
+                return new DelayedTimeValue(asWhenTime);
+            } catch (Exception e1) {
+            }
+            try {
+                ClockTime asClockTime = ClockTime.fromString(node);
+                return new DelayedTimeValue(asClockTime);
+            } catch (Exception e1) {
+            }
+
+            throw new ObjectMappingException("Couldn't find a delayedtimevalue amidst the string");
+        }
+
+        if (value.getValue() instanceof Map) {
+            Map node = value.getChildrenMap();
+            if (node.containsKey("type") && node.containsKey("statement")) {
+                Object typeObject = node.get("type");
+                Object statementObject = node.get("statement");
+                if (!(typeObject instanceof SimpleConfigurationNode && statementObject instanceof
+                        SimpleConfigurationNode)) {
+                    throw new IllegalArgumentException("Failed to parse map type/statement");
+                }
+                SimpleConfigurationNode asUnparsedType = (SimpleConfigurationNode) typeObject;
+                SimpleConfigurationNode asUnparsedStatement = (SimpleConfigurationNode) statementObject;
+                try {
+                    String asStringType = asUnparsedType.getString();
+                    String asStringStatement = asUnparsedStatement.getString();
+                    DelayedTimeType dtt = DelayedTimeType.fromString(asStringType);
```

Comment:
Yep, I actually start off making everything null, but missed changing them to optionals (because those are better!). Looks like I missed that here, though. I'll go ahead and change that.